gradle.properties

```diff
@@ -27,6 +27,7 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx1536m
```

Comment:
That's what I initially thought as well. However, the latest Android Gradle plugin will output a warning that dex will run out of process (causing a slower build), if Xmx is lower than 1536 MB (the default is 1024 MB). If you generate a new Android project with the latest Android Studio 2.2, the default `gradle.properties` will already have this value set to 1536 MB.